src/emqx_web_hook_actions.erl

```diff
@@ -42,7 +42,13 @@
                         title => #{en => <<"Request Method">>,
                                    zh => <<"请求方法"/utf8>>},
                         description => #{en => <<"Request Method. Note that the payload_template will be discarded in case of GET method">>,
-                                         zh => <<"请求方法。注意：当请求方法为 GET 的时候，payload_template 参数会被忽略"/utf8>>}}
+                                         zh => <<"请求方法。注意：当请求方法为 GET 的时候，payload_template 参数会被忽略"/utf8>>}},
+            path => #{type => string,
+                      required => false,
+                      default => <<>>,
+                      title => #{en => <<"Path">>,
+                                 zh => <<"Path">>},
+                      description => #{en => <<"A path component, variable interpolation from SQL statement is supported. This value will be concatenated with Request URL.">>}}
```

Comment:
Good idea, but it would be nice putting this `path` as a parameter of the Action, not resource. So that we could create more than one actions (with different path params) that use the same web-hook source.

Reply:
That sounds better. Thanks

Reply:
Fixed